frontend/web/components/IntegrationList.js

```diff
@@ -266,18 +266,20 @@ class IntegrationList extends Component {
           !this.state.isLoading &&
           this.state.activeIntegrations &&
           integrationList ? (
-            this.props.integrations.map((i, index) => (
-              <Integration
-                addIntegration={this.addIntegration}
-                editIntegration={this.editIntegration}
-                removeIntegration={this.removeIntegration}
-                projectId={this.props.projectId}
-                id={i}
-                key={i}
-                activeIntegrations={this.state.activeIntegrations[index]}
-                integration={integrationList[i]}
-              />
-            ))
+            this.props.integrations.map((i, index) =>
+              integrationList[i] ? (
```

Comment:
I guess this is just to protect against someone adding a key to the `integration_data` list with a null / empty value? Maybe there's a specific use case for this, but it seems a little unnecessary to me. If we're going down this line, it would make more sense to me to do some actual validation against a schema of sorts for `integrationList[i]` but I feel the chance of someone adding a falsy value here is pretty slim.

Reply:
Yeah actually this PR protects against the more common scenario where theres a key in the integrations feature that didn't exist in integration_data

Reply:
I thought that this PR would make that case completely redundant though, right, because we're not using the `integrations` feature at all?